src/common/model_reader.py

```diff
@@ -86,6 +86,7 @@ def __init__(self, yaml_files, data_common, version=DEFAULT_VERSION):
                 raise
 
         self.model.update({MODEL_SOURCE: model_file_src})
+        self.model.update({LIST_DELIMITER_PROP: delimiter if delimiter else ','})
```

Comment:
The default delimiter should be pipe character "|"

Reply:
Updated